From 50cc1fdcd00ff0c12426a1a014bd3dfe6fc0c3f5 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Fri, 7 Oct 2016 12:37:09 +0100
Subject: [PATCH 109/212] scripts/mkknlimg: Change string for DDT detection

The old "of_overlay_apply" string does not appear in 4.8 kernel builds.
"of_cfs_init" is both present and a more accurate indication of support
for dynamically loading overlays (although the trailer is now of little
practical use and the firmware assumes DT support and will look for
both .dtbo and -overlay.dtb overlays).

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 scripts/mkknlimg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mkknlimg b/scripts/mkknlimg
index 78c5845..a21f7e3 100755
--- a/scripts/mkknlimg
+++ b/scripts/mkknlimg
@@ -77,7 +77,7 @@ my $wanted_strings =
 	'brcm,bcm2709' => FLAG_PI | FLAG_DTOK | FLAG_270X,
 	'brcm,bcm2835' => FLAG_PI | FLAG_DTOK | FLAG_283X,
 	'brcm,bcm2836' => FLAG_PI | FLAG_DTOK | FLAG_283X,
-	'of_overlay_apply' => FLAG_DTOK | FLAG_DDTK,
+	'of_cfs_init' => FLAG_DTOK | FLAG_DDTK,
 };
 
 my $res = try_extract($kernel_file, $tmpfile1);
-- 
1.9.1

